MathGame.JsPeanut/MathGame.JsPeanut/Program.cs

```diff
@@ -0,0 +1,320 @@
+﻿using System;
+
+class Program
+{
+
+    public static void Main(string[] args)
+    {
+        List<Operation> previousGames = new List<Operation>();
+
+        bool doUserStillWantToPlay = true;
+
+        Console.WriteLine("Hello there! Welcome to this math game.");
+        Console.WriteLine("Available commands: -Score -Menu");
+        Console.WriteLine("Do you want to play?");
+        Console.WriteLine("Type 'yes' if so. If not, type 'exit'.");
+
+        while (doUserStillWantToPlay)
+        {
+            switch (Console.ReadLine())
+            {
+                case "yes":
+                    StartGame();
+                    break;
+                case "exit":
+                    doUserStillWantToPlay = false;
+                    break;
+            }
+        }
+
+        void StartGame()
+        {
+            Random rnd = new Random();
+            int a = rnd.Next(1, 100);
+            int b = rnd.Next(1, 100);
+
+            int Add(int x, int y)
+            {
+                return x + y;
+            }
+
+            int Substract(int x, int y)
+            {
+                return x - y;
+            }
+
+            int Multiply(int x, int y)
+            {
+                return x * y;
+            }
+
+            int Divide(int x, int y)
+            {
+                return x / y;
+            }
+
+            void Menu()
+            {
+                Console.WriteLine("Math Game is a game in which you can choose one of the 4 basic operations, and you will have to solve a calc with that operation.");
+                Console.WriteLine();
+                Console.WriteLine("Choose a math operation");
+                Console.WriteLine();
+                Console.WriteLine("Addition (Type A)");
+                Console.WriteLine("Substraction (Type S)");
+                Console.WriteLine("Multiplication (Type M)");
+                Console.WriteLine("Division (Type D)");
+            }
+
+            void AdditionGame()
+            {
+                Operation sum = new Operation();
```

Comment:
We can shorten this to the following: 

```suggestion
                Operation sum = new();
```